netsim/ansible/templates/bgp/dellos10.macro.j2

```diff
@@ -66,7 +66,11 @@
  exit
 {%   endif %}
 {% endif %}
-  
+
+{# Override default timers of 60/180 #}
+  timers 3 9
```

Comment:
This was unnecessary, as the BGP timers impact only the session failure detection logic.

Reply:
Yes, and they are negotiated so when peering with FRR things work fine. However, between Dell OS10 devices they would use defaults less suited to data center environments